workbench_access.module

```diff
@@ -233,12 +233,12 @@ function workbench_access_element_info_alter(array &$info) {
 }
 
 /**
- * Process callback.
+ * Process callback for inline entity form.
  */
 function workbench_access_process_inline_entity_form(array &$element, FormStateInterface $form_state, &$complete_form) {
   $entity = $element['#entity'];
   if (!$entity instanceof ContentEntityInterface) {
     return $element;
   }
-  return \Drupal::classResolver()->getInstanceFromDefinition(FormAlterHelper::class)->alterForm($element, $complete_form, $entity);
+  return \Drupal::classResolver()->getInstanceFromDefinition(FormAlterHelper::class)->alterForm($element, $complete_form, $form_state);
```

Comment:
This won't work - the $entity for an inline entity form comes from `$element['#entity']`.

Using `$form_state->getFormObject()->getEntity()` in this instance, will get the entity of the parent form, not the inline form.

I don't think we can do this without retaining the entity parameter.

So perhaps we need both form state and entity as parameters

Or we just mark this as won't fix - depends i the plugins need the form state

Reply:
Right now, none of the plugins need the form state. Things like IEF were why I thought we might need it.

Reply:
so I guess the question is do we think it will be needed

cause adding it later will be an api break

if there's a 1% chance, we should do it before we get to beta

Reply:
I think there is a 1% chance, so I suppose we change the current signature to pass both $form_state and an optional $entity.